rfc/2019-12-11-Talent-Plan-Courses.md

```diff
@@ -0,0 +1,79 @@
+
+## Course Framework
+
+![course map](../media/rfc-talent-plan-courses.png)
+
+The picture above shows the latest framework of Talent Plan Courses. In addition to TiDB related learning materials, lots of other sections also will be provided. Following are the details:
+
+- Open Source Cooperation
+
+  Open source spirit is one of the cores of the TiDB Community. We hope more people embrace open source,so courses about it is a significantly important part of this course system. At current, multiple courses are on the list:
+
+  - Introduction to OSS
+    To let people know about open-source spirit, basic knowledge such as the mean of open-source spirit and its history will be covered.
+
+  - How to use Git&Github
+    Git is a version control software that is broadly used to accelerate open source development, and Github is a popular website for people around the world to participate in open source project development which based on Git. After taking this course, participants should know the usage of Git and Github.
+
+  - Open Source Practice in TiDB
+    TiDB ecosystem is made up of multiple projects and all open-sourced on Github. For people who want to contribute to TiDB, open-source practice in TiDB would be helpful for them.
+
+- Programming Language
+
+  For anyone who wants to participate in an open-source project, knows the project's development programming language is the most basic requirement. We will deliver some courses about programming language to help participant learns it quickly.
+
+  - Golang
+    The Go programming language is heavily used in industry, also the TiDB ecosystem. We advise everyone to learn a bit about it:
+
+  - [A Tour Of Golang](https://tour.golang.org/) is recommanded since it's a great official tutorial for the newbie to learn the basic language features.
+  - [The Go Programming Language](http://www.gopl.io/) is recommended as a reference book for people want to learn the language in detail.
+  - [Talent Plan MapReduce](https://github.com/pingcap/talent-plan/tree/master/tidb/mapreduce) will be a great tool for participants to judge their learning outcomes.
+
+  - Rust
+    The Rust programming language is becoming more and more popular since it has many advanced features. For people interested in TiKV which is based on rust, we recommend the following materials:
+
+  - [Learn Rust](https://www.rust-lang.org/learn) is official study materials, it contains a book, an interactive practice assignment, and some code examples. We believe anyone interested in the Rust programming language can benefit from it.
+  - [Practical Networked Application in Rust](https://github.com/pingcap/talent-plan/tree/master/rust) is a programming assignment framework develop by [Brian Anderson](https://github.com/brson), who was a core team member of the rust community. We strongly suggest taking this assignment to consolidate rust programming skills.
```

Comment:
```suggestion
  - [Practical Networked Application in Rust](https://github.com/pingcap/talent-plan/tree/master/rust) - A series of projects that incrementally develop a single Rust project from the ground up into a high-performance, networked, parallel and asynchronous key/value store. Along the way, various real-world and practical Rust development subject matter are explored and discussed. This course is developed by [Brian Anderson](https://github.com/brson), who was one of the core writers of the Rust language.
```

Reply:
A suggestion - maybe you should just copy the original introduction for each item on their specific website, instead of writing all of them again on your own.